### PR TITLE
Home Updates

### PR DESCRIPTION
**Pull request by RevolutionTech**

### CODE REVIEW

File: artist/forms.py [hunk 1]
```diff
@@ -12,8 +12,12 @@
 class ArtistApplyForm(forms.Form):
```
Comment (RevolutionTech) [inline artist/forms.py]: Looks like the changes on this page should have been applied to https://github.com/RevolutionTech/perdiem-django/pull/363. Maybe you should apply the changes you've made here onto that PR and then we can revert this file.

File: templates/artist/artist_application.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h3 class="extra-padding">Artist Application</h3>
+    <h3 class="extra-padding">Create An Artist Account</h3>
```
Comment (RevolutionTech) [inline templates/artist/artist_application.html]: Should this change also be in https://github.com/RevolutionTech/perdiem-django/pull/363? After you apply that change to the other PR I can revert this file.

File: templates/base.html [hunk 3]
```diff
@@ -166,7 +166,8 @@
             <ul class="footer-items">
                 <li><a href="{% url 'privacy' %}">Privacy</a></li>
                 <li><a href="{% url 'terms' %}">Terms of Use</a></li>
-                <li><a href="{% url 'resources' %}">For Artists</a></li>
+                <li><a href="{% url 'resources' %}">Artists</a></li>
```
Comment (RevolutionTech) [inline templates/base.html]: Should this page be renamed to `artist-resources`?

File: artist/forms.py (hunk 1 above)
Comment (localastronaut) [reply artist/forms.py]: Made those updates so this should be good to be reverted.

**Pull request merged**